### PR TITLE
Reading thermal_zone temp sensor data in getTemps

### PR DESCRIPTION
**Pull request by chraac**
It'll enumerate /sys/class/thermal/thermal_zone* to collect sensor datas

### CODE REVIEW

File: src/Linfo/OS/Linux.php [hunk 1]
```diff
@@ -562,6 +562,40 @@ public function getTemps()
...
+                // Temperatures
+                if (is_file($labelpath)) {
+                    $label = Common::getContents($labelpath);
+                    $value = Common::getIntFromFile($valuepath);
+                    $value /= $value > 10000 ? 1000 : 1;
+                    $unit = 'C'; // I don't think this is ever going to be in F
+                }
+
+                // Append values
+                $thermal_zone_vals[] = array(
```
Comment (jrgp) [inline src/Linfo/OS/Linux.php]: Can you move the $thermal_zone_vals append to inside the is_file block above? That way if the file doesn't exist we won't be appending missing variables.

Comment (chraac) [reply src/Linfo/OS/Linux.php]: okay

**Comment by jrgp** [comment]
merged!

**Comment by chraac** [comment]
thanks!

**Pull request merged**